packages/venia-ui/lib/tokens.css

```diff
@@ -0,0 +1,136 @@
+/* global tokens */
+:global(:root) {
```

Comment:
@jimbo I saw in the JIRA ticket that the first task was to copy these from the styleguide to venia-ui. Was the wording intentional? This is an opportunity for the guide and venia-ui to get out of sync. Should we be using tokens as an export of styleguide rather than a copy?

Reply:
That's correct. They could be out of sync, but `venia-styleguide` is not intended to be a dependency of `venia-ui` in any way. It shouldn't export anything for consumption.